src/components/dialogs/error.vue

```diff
@@ -58,6 +67,10 @@ export default class ErrorDialog extends Vue {
   async hideModal () {
     await ErrorModule.clearAppErrors()
   }
+  get isExists () {
+    const errors = ErrorModule[ErrorTypes.GET_ERRORS]
+    return errors[0] && errors[0].id === 'entry-already-exists'
+  }
```

Comment:
This looks fine, but please follow existing code style (blank line between functions).